src/Sequence.cc

```diff
@@ -388,10 +388,10 @@ Sequence::execute_sequence_impl(Iterator step_begin, Iterator step_end, Context&
 {
     Iterator step = step_begin;
 
-    if (comm and comm->immediate_termination_requested_)
-        throw Error{ gul14::cat(abort_marker, "Stop on user request") };
     while (step < step_end)
     {
+        if (comm and comm->immediate_termination_requested_)
```

Comment:
Ok. The check for a terminating request is now done more frequently.

This is ok but I ask myself what happens when the previous steps takes a time before it finishes and parallel another thread calls `Executor::chancel()`. The sequence will only recognise the `immediate_termination_requested_` before the next step executes and exit gracefully. But since we have a Lua hook that checks every 100 instructions for this 'flag' it is ok and the custom Lua `sleep` function also checks it. Therefore: 👍